packages/walletconnect/src/provider.ts

```diff
@@ -218,9 +207,12 @@ export class WalletConnectProvider extends SignerProvider {
 
   // ---------- Private ----------------------------------------------- //
 
-  private cleanHistory() {
+  private cleanHistory(checkResponse: boolean) {
```

Comment:
Sometimes there are multiple requests at the same time. We need to filter out those requests that have not received a response, otherwise walletconnect will throw an exception during processing.